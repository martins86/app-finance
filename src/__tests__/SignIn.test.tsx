import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';

import SignIn from '../pages/sign-in';

describe('Test <SignIn />', () => {
  test('Should match snapshot', () => {
    // Arrange
    const { container } = render(
      <Router>
        <SignIn />
      </Router>
    );

    // Act & Assert
    expect(container).toMatchSnapshot();
  });
});
