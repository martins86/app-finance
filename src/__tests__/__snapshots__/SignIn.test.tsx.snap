// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <SignIn /> Should match snapshot 1`] = `
<div>
  <div
    class="sign-in"
  >
    <label
      class="label-system"
    >
      SISTEMA DE LOGIN
    </label>
    <div
      class="sign-in-content"
    >
      <input
        class="input-primary"
        placeholder="Digite seu E-mail"
        type="email"
        value=""
      />
      <input
        class="input-primary"
        placeholder="Digite sua Senha"
        type="password"
        value=""
      />
      <label
        class="label-error"
      />
      <button
        class="btn btn--primary btn--full-width"
        data-testid="button"
        title="Entrar"
        type="button"
      >
        Entrar
      </button>
      <label
        class="label-sign"
      >
        NÃ£o tem uma conta?
        <strong
          class="text-strong-link"
        >
          <a
            href="/app-finance/sign-up"
          >
            Registre-se
          </a>
        </strong>
      </label>
    </div>
  </div>
</div>
`;
